util_test.go

```diff
@@ -0,0 +1,49 @@
+package kazaam
+
+import (
+	"testing"
+)
+
+type IsJsonTest struct {
+	json string
+	valid   bool
+}
+
+var jsonTests = []IsJsonTest{
```

Comment:
Could you add a few tests for some other cases like:
- numeric values
- boolean/nil values
- json that is a top-level array (i.e. `["element one", "element 2"]` etc